src/features/featureincludeallnetworks.h

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef FEATURE_INCLUDEALLNETWORKS_H
```

Comment:
I would call it "FeatureIOSKillSwitch". `IncludeAllNetworks` is just the API that allows us to implement the kill-switch.